src/GeneralStateTestsFiller/VMTests/vmArithmeticTest/divByZeroFiller.yml

```diff
@@ -0,0 +1,230 @@
+divByZero:
+
+  # A standard location for division by zero tests.
+  #
+  # Opcodes where this is relevant:
+  #   DIV
+  #   SDIV
+  #   MOD
+  #   SMOD
+  #   ADDMOD
+  #   MULMOD
+  #
+  # Any division or mod by zero returns zero.
+
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentGasLimit: 100000000
+    currentNumber: 1
+    currentTimestamp: 1000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  _info:
+    comment: Ori Pomerantz qbzzt1@gmail.com
+
+  pre:
+
+    # The first parameter is the opcode:
+    #  0 - DIV
+    #  1 - SDIV
+    #  2 - MOD
+    #  3 - SMOD
+    #  4 - ADDMOD
+    #  5 - MULMOD
+    #
+    # The other parameters are the parameters for the opcode
+    cccccccccccccccccccccccccccccccccccccccc:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code: |
+        {  
+            (def 'NOP 0)
+            (def 'opcode $4)
+            (def 'a      $36)
+            (def 'b      $68)
+
+            (if (= opcode 0x04) [[0]] (div a 0) NOP)
+            (if (= opcode 0x05) [[0]] (sdiv a 0) NOP)
+            (if (= opcode 0x06) [[0]] (mod a 0) NOP)
+            (if (= opcode 0x07) [[0]] (smod a 0) NOP)
+            (if (= opcode 0x08) [[0]] (addmod a b 0) NOP)
+            (if (= opcode 0x09) [[0]] (mulmod a b 0) NOP)
+        }
+      nonce: '0'
+      storage: {}
+
+
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code: '0x'
+      nonce: '0'
+      storage: {}
+      
+
+
+
+  transaction:
+    data:
+
+    # DIV tests  (opcode 0x04)
+    - :label div_2_0       :abi f(uint,uint) 0x04 2
+    - :label div_1_0       :abi f(uint,uint) 0x04 1
+    - :label div_0_0       :abi f(uint,uint) 0x04 0
+    - :label div_neg1_0    :abi f(uint,uint) 0x04 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
```

Comment:
please can you double check vmtrace that it is indeed running an opcode 0x04 with 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF as an argument ?

Reply:
OK, done. I ran this command:

```
./dretesteth.sh -t GeneralStateTests/VMTests/vmArithmeticTest -- --testpath /home/qbzzt1/mergers/tests --singletest divByZero --filltests --all -d 3 --vmtraceraw |& tee x
```

The relevant line is line 23. Here it is (with a few newlines for clarity):
```
"stack":["0x0",
         "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"],
"returnStack":null,"returnData":"0x","depth":1,"refund":0,
"opName":"DIV","error":""}
```

The following line shows that the result is the expected zero:
```
{"pc":22,"op":96,"gas":"0x4c45f04","gasCost":"0x3","memory":"0x","memSize":0,
"stack":["0x0"],"returnStack":null,"returnData":"0x","depth":1,"refund":0,
"opName":"PUSH1","error":""}
```